examples/gno.land/r/gnoland/home/home.gno

```diff
@@ -30,144 +25,233 @@ func Render(_ string) string {
-- [Explore the Ecosystem](/ecosystem)
-- [Careers](https://jobs.ashbyhq.com/allinbits)
+- Reusability - smart contracts built with Gno are designed to encourage reusability, composability, and forkability when needed. 
+- Open and transparent - all smart contracts and applications are fully open-source and auditable, ensuring verifiability. 
+- Decentralized Governance – governance is maintained by trusted contributors and the community. 
+- Developer-Friendly – Go developers can easily transition to Gno and start building without learning a new, restrictive language.
 
-## Build with Gno
 
-- [Write Gno in the browser](https://play.gno.land)
-- [Read about the Gno Language](/gnolang)
-- [Visit the official documentation](https://docs.gno.land)
-- [Gno by Example](https://gno-by-example.com/)
-- [Efficient local development for Gno](https://docs.gno.land/gno-tooling/cli/gno-tooling-gnodev)
-- [Get testnet GNOTs](https://faucet.gno.land)
+---
 
-## Explore the universe
+Why Gno?
 
-- [Discover demo packages](https://github.com/gnolang/gno/tree/master/examples)
-- [Gnoscan](https://gnoscan.io)
-- [Portal Loop](https://docs.gno.land/concepts/portal-loop)
-- [Testnet 4](https://test4.gno.land/)
-- [Faucet Hub](https://faucet.gno.land)
+## A new decentralized operating system (dOS) to power the Internet of Everyone
 
-## [Latest Blogposts](/r/gnoland/blog)
+We are building an operating system and network for everyone who wants to contribute and change the way the human component of the web currently operates. Find out more how gno.land is making this happen. 
 
-:latest-blogposts:
+<gno-columns>
 
-## [Latest Events](/r/gnoland/events)
+####
 
-:upcoming-events:
+![Social coordination & community](#)
+
+### Social coordination & community
+
+Censorship-resistant, community-organized social impact platforms that provide coordination, conversation, conflict resolution, and a builder-owned governance sstructure. 
+
+
+####
+
+![Governance & organization](#)
+
+### Governance & Organization
+
+Redefining governance frameworks, negotiation systems, and decentralized autonomous organizations (DAOs) to prioritize a common ecosystem alignment, active participation and ownership, and application specific implementations in lieu of a solely capital-based control system.
+
+####
+
+![New-Economic-Model](#)
+
+### New Economic Model
+
+A contribution-driven economic model that incentivizes creators, innovators, moderators, and maintainers to ensure a sustainable, builder-aligned governance and organizational framework for an alternative model to the current speculation or advertising based internet. 
+
+####
+
+![The next killer app](#)
+
+### The next killer app
+
+A reusable technical architecture for everyone to build the killer applications of the future that attract the next wave of builders and users to the Internet of Everyone.   
 
-## [Hall of Fame](/r/leon/hof)
 
-:latest-hof:
+</gno-columns>
+
+TODO: Add 'Beta Mainnet Coming Soon!' section
```

Comment:
@leohhhn can you make this dynamic, i.e., by having like r/gnoland/blog to have a "news" tag?